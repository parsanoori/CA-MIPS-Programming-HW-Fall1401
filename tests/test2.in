7
329996248
254808882
332896409
223234617
538820899
261927638
402939283