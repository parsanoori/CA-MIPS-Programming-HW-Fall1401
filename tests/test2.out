223234617
254808882
261927638
329996248
332896409
402939283
538820899